internal/router/middlewares/ratelim.go

```diff
@@ -86,6 +87,7 @@ func RateLimitController(cfg RateLimiterConfig) (mux.MiddlewareFunc, error) {
 					_ = json.NewEncoder(w).Encode(errors.ServiceError{Message: "reading request body"})
 					return
 				}
+				log.Warn().Str("body", string(fullBody)).Msg("call to legacy RPC")
```

Comment:
Adding the log-line here since the rate-limiter middleware is the only place we already read the POST body.
So we can do this analysis for free.